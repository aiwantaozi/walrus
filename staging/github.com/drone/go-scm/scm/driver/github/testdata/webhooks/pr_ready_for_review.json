{
  "action": "ready_for_review",
  "number": 38,
  "pull_request": {
    "url": "https://api.github.com/repos/wings-software/meet-git-sync-test/pulls/38",
    "id": 1492580810,
    "node_id": "PR_kwDOHbOleM5Y9vnK",
    "html_url": "https://github.com/wings-software/meet-git-sync-test/pull/38",
    "diff_url": "https://github.com/wings-software/meet-git-sync-test/pull/38.diff",
    "patch_url": "https://github.com/wings-software/meet-git-sync-test/pull/38.patch",
    "issue_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/issues/38",
    "number": 38,
    "state": "open",
    "locked": false,
    "title": "Update dgdggd.me",
    "user": {
      "login": "rathodmeetsatish",
      "id": 84321134,
      "node_id": "MDQ6VXNlcjg0MzIxMTM0",
      "avatar_url": "https://avatars.githubusercontent.com/u/84321134?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/rathodmeetsatish",
      "html_url": "https://github.com/rathodmeetsatish",
      "followers_url": "https://api.github.com/users/rathodmeetsatish/followers",
      "following_url": "https://api.github.com/users/rathodmeetsatish/following{/other_user}",
      "gists_url": "https://api.github.com/users/rathodmeetsatish/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/rathodmeetsatish/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/rathodmeetsatish/subscriptions",
      "organizations_url": "https://api.github.com/users/rathodmeetsatish/orgs",
      "repos_url": "https://api.github.com/users/rathodmeetsatish/repos",
      "events_url": "https://api.github.com/users/rathodmeetsatish/events{/privacy}",
      "received_events_url": "https://api.github.com/users/rathodmeetsatish/received_events",
      "type": "User",
      "site_admin": false
    },
    "body": null,
    "created_at": "2023-08-28T19:15:53Z",
    "updated_at": "2023-08-28T19:16:06Z",
    "closed_at": null,
    "merged_at": null,
    "merge_commit_sha": "d427e98e038a88f10e57d48c34f287e3ad20a0bf",
    "assignee": null,
    "assignees": [

    ],
    "requested_reviewers": [

    ],
    "requested_teams": [

    ],
    "labels": [

    ],
    "milestone": null,
    "draft": false,
    "commits_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/pulls/38/commits",
    "review_comments_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/pulls/38/comments",
    "review_comment_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/pulls/comments{/number}",
    "comments_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/issues/38/comments",
    "statuses_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/statuses/212015fb011c49a2913c4a1a860ce07c0821c362",
    "head": {
      "label": "wings-software:rathodmeetsatish-patch-25",
      "ref": "rathodmeetsatish-patch-25",
      "sha": "212015fb011c49a2913c4a1a860ce07c0821c362",
      "user": {
        "login": "wings-software",
        "id": 18273000,
        "node_id": "MDEyOk9yZ2FuaXphdGlvbjE4MjczMDAw",
        "avatar_url": "https://avatars.githubusercontent.com/u/18273000?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/wings-software",
        "html_url": "https://github.com/wings-software",
        "followers_url": "https://api.github.com/users/wings-software/followers",
        "following_url": "https://api.github.com/users/wings-software/following{/other_user}",
        "gists_url": "https://api.github.com/users/wings-software/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/wings-software/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/wings-software/subscriptions",
        "organizations_url": "https://api.github.com/users/wings-software/orgs",
        "repos_url": "https://api.github.com/users/wings-software/repos",
        "events_url": "https://api.github.com/users/wings-software/events{/privacy}",
        "received_events_url": "https://api.github.com/users/wings-software/received_events",
        "type": "Organization",
        "site_admin": false
      },
      "repo": {
        "id": 498312568,
        "node_id": "R_kgDOHbOleA",
        "name": "meet-git-sync-test",
        "full_name": "wings-software/meet-git-sync-test",
        "private": true,
        "owner": {
          "login": "wings-software",
          "id": 18273000,
          "node_id": "MDEyOk9yZ2FuaXphdGlvbjE4MjczMDAw",
          "avatar_url": "https://avatars.githubusercontent.com/u/18273000?v=4",
          "gravatar_id": "",
          "url": "https://api.github.com/users/wings-software",
          "html_url": "https://github.com/wings-software",
          "followers_url": "https://api.github.com/users/wings-software/followers",
          "following_url": "https://api.github.com/users/wings-software/following{/other_user}",
          "gists_url": "https://api.github.com/users/wings-software/gists{/gist_id}",
          "starred_url": "https://api.github.com/users/wings-software/starred{/owner}{/repo}",
          "subscriptions_url": "https://api.github.com/users/wings-software/subscriptions",
          "organizations_url": "https://api.github.com/users/wings-software/orgs",
          "repos_url": "https://api.github.com/users/wings-software/repos",
          "events_url": "https://api.github.com/users/wings-software/events{/privacy}",
          "received_events_url": "https://api.github.com/users/wings-software/received_events",
          "type": "Organization",
          "site_admin": false
        },
        "html_url": "https://github.com/wings-software/meet-git-sync-test",
        "description": null,
        "fork": false,
        "url": "https://api.github.com/repos/wings-software/meet-git-sync-test",
        "forks_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/forks",
        "keys_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/keys{/key_id}",
        "collaborators_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/collaborators{/collaborator}",
        "teams_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/teams",
        "hooks_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/hooks",
        "issue_events_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/issues/events{/number}",
        "events_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/events",
        "assignees_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/assignees{/user}",
        "branches_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/branches{/branch}",
        "tags_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/tags",
        "blobs_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/git/blobs{/sha}",
        "git_tags_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/git/tags{/sha}",
        "git_refs_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/git/refs{/sha}",
        "trees_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/git/trees{/sha}",
        "statuses_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/statuses/{sha}",
        "languages_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/languages",
        "stargazers_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/stargazers",
        "contributors_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/contributors",
        "subscribers_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/subscribers",
        "subscription_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/subscription",
        "commits_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/commits{/sha}",
        "git_commits_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/git/commits{/sha}",
        "comments_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/comments{/number}",
        "issue_comment_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/issues/comments{/number}",
        "contents_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/contents/{+path}",
        "compare_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/compare/{base}...{head}",
        "merges_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/merges",
        "archive_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/{archive_format}{/ref}",
        "downloads_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/downloads",
        "issues_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/issues{/number}",
        "pulls_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/pulls{/number}",
        "milestones_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/milestones{/number}",
        "notifications_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/notifications{?since,all,participating}",
        "labels_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/labels{/name}",
        "releases_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/releases{/id}",
        "deployments_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/deployments",
        "created_at": "2022-05-31T11:47:10Z",
        "updated_at": "2022-05-31T11:47:35Z",
        "pushed_at": "2023-08-28T19:15:54Z",
        "git_url": "git://github.com/wings-software/meet-git-sync-test.git",
        "ssh_url": "git@github.com:wings-software/meet-git-sync-test.git",
        "clone_url": "https://github.com/wings-software/meet-git-sync-test.git",
        "svn_url": "https://github.com/wings-software/meet-git-sync-test",
        "homepage": null,
        "size": 171,
        "stargazers_count": 0,
        "watchers_count": 0,
        "language": "Roff",
        "has_issues": true,
        "has_projects": true,
        "has_downloads": true,
        "has_wiki": true,
        "has_pages": false,
        "has_discussions": false,
        "forks_count": 0,
        "mirror_url": null,
        "archived": false,
        "disabled": false,
        "open_issues_count": 23,
        "license": null,
        "allow_forking": false,
        "is_template": false,
        "web_commit_signoff_required": false,
        "topics": [

        ],
        "visibility": "private",
        "forks": 0,
        "open_issues": 23,
        "watchers": 0,
        "default_branch": "main",
        "allow_squash_merge": true,
        "allow_merge_commit": true,
        "allow_rebase_merge": true,
        "allow_auto_merge": false,
        "delete_branch_on_merge": false,
        "allow_update_branch": false,
        "use_squash_pr_title_as_default": false,
        "squash_merge_commit_message": "COMMIT_MESSAGES",
        "squash_merge_commit_title": "COMMIT_OR_PR_TITLE",
        "merge_commit_message": "PR_TITLE",
        "merge_commit_title": "MERGE_MESSAGE"
      }
    },
    "base": {
      "label": "wings-software:main",
      "ref": "main",
      "sha": "8b8d2b3e6a2ee6df108c8429e86c9a750e728f9d",
      "user": {
        "login": "wings-software",
        "id": 18273000,
        "node_id": "MDEyOk9yZ2FuaXphdGlvbjE4MjczMDAw",
        "avatar_url": "https://avatars.githubusercontent.com/u/18273000?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/wings-software",
        "html_url": "https://github.com/wings-software",
        "followers_url": "https://api.github.com/users/wings-software/followers",
        "following_url": "https://api.github.com/users/wings-software/following{/other_user}",
        "gists_url": "https://api.github.com/users/wings-software/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/wings-software/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/wings-software/subscriptions",
        "organizations_url": "https://api.github.com/users/wings-software/orgs",
        "repos_url": "https://api.github.com/users/wings-software/repos",
        "events_url": "https://api.github.com/users/wings-software/events{/privacy}",
        "received_events_url": "https://api.github.com/users/wings-software/received_events",
        "type": "Organization",
        "site_admin": false
      },
      "repo": {
        "id": 498312568,
        "node_id": "R_kgDOHbOleA",
        "name": "meet-git-sync-test",
        "full_name": "wings-software/meet-git-sync-test",
        "private": true,
        "owner": {
          "login": "wings-software",
          "id": 18273000,
          "node_id": "MDEyOk9yZ2FuaXphdGlvbjE4MjczMDAw",
          "avatar_url": "https://avatars.githubusercontent.com/u/18273000?v=4",
          "gravatar_id": "",
          "url": "https://api.github.com/users/wings-software",
          "html_url": "https://github.com/wings-software",
          "followers_url": "https://api.github.com/users/wings-software/followers",
          "following_url": "https://api.github.com/users/wings-software/following{/other_user}",
          "gists_url": "https://api.github.com/users/wings-software/gists{/gist_id}",
          "starred_url": "https://api.github.com/users/wings-software/starred{/owner}{/repo}",
          "subscriptions_url": "https://api.github.com/users/wings-software/subscriptions",
          "organizations_url": "https://api.github.com/users/wings-software/orgs",
          "repos_url": "https://api.github.com/users/wings-software/repos",
          "events_url": "https://api.github.com/users/wings-software/events{/privacy}",
          "received_events_url": "https://api.github.com/users/wings-software/received_events",
          "type": "Organization",
          "site_admin": false
        },
        "html_url": "https://github.com/wings-software/meet-git-sync-test",
        "description": null,
        "fork": false,
        "url": "https://api.github.com/repos/wings-software/meet-git-sync-test",
        "forks_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/forks",
        "keys_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/keys{/key_id}",
        "collaborators_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/collaborators{/collaborator}",
        "teams_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/teams",
        "hooks_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/hooks",
        "issue_events_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/issues/events{/number}",
        "events_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/events",
        "assignees_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/assignees{/user}",
        "branches_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/branches{/branch}",
        "tags_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/tags",
        "blobs_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/git/blobs{/sha}",
        "git_tags_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/git/tags{/sha}",
        "git_refs_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/git/refs{/sha}",
        "trees_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/git/trees{/sha}",
        "statuses_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/statuses/{sha}",
        "languages_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/languages",
        "stargazers_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/stargazers",
        "contributors_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/contributors",
        "subscribers_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/subscribers",
        "subscription_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/subscription",
        "commits_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/commits{/sha}",
        "git_commits_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/git/commits{/sha}",
        "comments_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/comments{/number}",
        "issue_comment_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/issues/comments{/number}",
        "contents_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/contents/{+path}",
        "compare_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/compare/{base}...{head}",
        "merges_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/merges",
        "archive_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/{archive_format}{/ref}",
        "downloads_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/downloads",
        "issues_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/issues{/number}",
        "pulls_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/pulls{/number}",
        "milestones_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/milestones{/number}",
        "notifications_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/notifications{?since,all,participating}",
        "labels_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/labels{/name}",
        "releases_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/releases{/id}",
        "deployments_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/deployments",
        "created_at": "2022-05-31T11:47:10Z",
        "updated_at": "2022-05-31T11:47:35Z",
        "pushed_at": "2023-08-28T19:15:54Z",
        "git_url": "git://github.com/wings-software/meet-git-sync-test.git",
        "ssh_url": "git@github.com:wings-software/meet-git-sync-test.git",
        "clone_url": "https://github.com/wings-software/meet-git-sync-test.git",
        "svn_url": "https://github.com/wings-software/meet-git-sync-test",
        "homepage": null,
        "size": 171,
        "stargazers_count": 0,
        "watchers_count": 0,
        "language": "Roff",
        "has_issues": true,
        "has_projects": true,
        "has_downloads": true,
        "has_wiki": true,
        "has_pages": false,
        "has_discussions": false,
        "forks_count": 0,
        "mirror_url": null,
        "archived": false,
        "disabled": false,
        "open_issues_count": 23,
        "license": null,
        "allow_forking": false,
        "is_template": false,
        "web_commit_signoff_required": false,
        "topics": [

        ],
        "visibility": "private",
        "forks": 0,
        "open_issues": 23,
        "watchers": 0,
        "default_branch": "main",
        "allow_squash_merge": true,
        "allow_merge_commit": true,
        "allow_rebase_merge": true,
        "allow_auto_merge": false,
        "delete_branch_on_merge": false,
        "allow_update_branch": false,
        "use_squash_pr_title_as_default": false,
        "squash_merge_commit_message": "COMMIT_MESSAGES",
        "squash_merge_commit_title": "COMMIT_OR_PR_TITLE",
        "merge_commit_message": "PR_TITLE",
        "merge_commit_title": "MERGE_MESSAGE"
      }
    },
    "_links": {
      "self": {
        "href": "https://api.github.com/repos/wings-software/meet-git-sync-test/pulls/38"
      },
      "html": {
        "href": "https://github.com/wings-software/meet-git-sync-test/pull/38"
      },
      "issue": {
        "href": "https://api.github.com/repos/wings-software/meet-git-sync-test/issues/38"
      },
      "comments": {
        "href": "https://api.github.com/repos/wings-software/meet-git-sync-test/issues/38/comments"
      },
      "review_comments": {
        "href": "https://api.github.com/repos/wings-software/meet-git-sync-test/pulls/38/comments"
      },
      "review_comment": {
        "href": "https://api.github.com/repos/wings-software/meet-git-sync-test/pulls/comments{/number}"
      },
      "commits": {
        "href": "https://api.github.com/repos/wings-software/meet-git-sync-test/pulls/38/commits"
      },
      "statuses": {
        "href": "https://api.github.com/repos/wings-software/meet-git-sync-test/statuses/212015fb011c49a2913c4a1a860ce07c0821c362"
      }
    },
    "author_association": "COLLABORATOR",
    "auto_merge": null,
    "active_lock_reason": null,
    "merged": false,
    "mergeable": true,
    "rebaseable": true,
    "mergeable_state": "clean",
    "merged_by": null,
    "comments": 0,
    "review_comments": 0,
    "maintainer_can_modify": false,
    "commits": 1,
    "additions": 4,
    "deletions": 1,
    "changed_files": 1
  },
  "repository": {
    "id": 498312568,
    "node_id": "R_kgDOHbOleA",
    "name": "meet-git-sync-test",
    "full_name": "wings-software/meet-git-sync-test",
    "private": true,
    "owner": {
      "login": "wings-software",
      "id": 18273000,
      "node_id": "MDEyOk9yZ2FuaXphdGlvbjE4MjczMDAw",
      "avatar_url": "https://avatars.githubusercontent.com/u/18273000?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/wings-software",
      "html_url": "https://github.com/wings-software",
      "followers_url": "https://api.github.com/users/wings-software/followers",
      "following_url": "https://api.github.com/users/wings-software/following{/other_user}",
      "gists_url": "https://api.github.com/users/wings-software/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/wings-software/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/wings-software/subscriptions",
      "organizations_url": "https://api.github.com/users/wings-software/orgs",
      "repos_url": "https://api.github.com/users/wings-software/repos",
      "events_url": "https://api.github.com/users/wings-software/events{/privacy}",
      "received_events_url": "https://api.github.com/users/wings-software/received_events",
      "type": "Organization",
      "site_admin": false
    },
    "html_url": "https://github.com/wings-software/meet-git-sync-test",
    "description": null,
    "fork": false,
    "url": "https://api.github.com/repos/wings-software/meet-git-sync-test",
    "forks_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/forks",
    "keys_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/keys{/key_id}",
    "collaborators_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/collaborators{/collaborator}",
    "teams_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/teams",
    "hooks_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/hooks",
    "issue_events_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/issues/events{/number}",
    "events_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/events",
    "assignees_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/assignees{/user}",
    "branches_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/branches{/branch}",
    "tags_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/tags",
    "blobs_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/git/blobs{/sha}",
    "git_tags_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/git/tags{/sha}",
    "git_refs_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/git/refs{/sha}",
    "trees_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/git/trees{/sha}",
    "statuses_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/statuses/{sha}",
    "languages_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/languages",
    "stargazers_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/stargazers",
    "contributors_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/contributors",
    "subscribers_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/subscribers",
    "subscription_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/subscription",
    "commits_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/commits{/sha}",
    "git_commits_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/git/commits{/sha}",
    "comments_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/comments{/number}",
    "issue_comment_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/issues/comments{/number}",
    "contents_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/contents/{+path}",
    "compare_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/compare/{base}...{head}",
    "merges_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/merges",
    "archive_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/{archive_format}{/ref}",
    "downloads_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/downloads",
    "issues_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/issues{/number}",
    "pulls_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/pulls{/number}",
    "milestones_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/milestones{/number}",
    "notifications_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/notifications{?since,all,participating}",
    "labels_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/labels{/name}",
    "releases_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/releases{/id}",
    "deployments_url": "https://api.github.com/repos/wings-software/meet-git-sync-test/deployments",
    "created_at": "2022-05-31T11:47:10Z",
    "updated_at": "2022-05-31T11:47:35Z",
    "pushed_at": "2023-08-28T19:15:54Z",
    "git_url": "git://github.com/wings-software/meet-git-sync-test.git",
    "ssh_url": "git@github.com:wings-software/meet-git-sync-test.git",
    "clone_url": "https://github.com/wings-software/meet-git-sync-test.git",
    "svn_url": "https://github.com/wings-software/meet-git-sync-test",
    "homepage": null,
    "size": 171,
    "stargazers_count": 0,
    "watchers_count": 0,
    "language": "Roff",
    "has_issues": true,
    "has_projects": true,
    "has_downloads": true,
    "has_wiki": true,
    "has_pages": false,
    "has_discussions": false,
    "forks_count": 0,
    "mirror_url": null,
    "archived": false,
    "disabled": false,
    "open_issues_count": 23,
    "license": null,
    "allow_forking": false,
    "is_template": false,
    "web_commit_signoff_required": false,
    "topics": [

    ],
    "visibility": "private",
    "forks": 0,
    "open_issues": 23,
    "watchers": 0,
    "default_branch": "main"
  },
  "organization": {
    "login": "wings-software",
    "id": 18273000,
    "node_id": "MDEyOk9yZ2FuaXphdGlvbjE4MjczMDAw",
    "url": "https://api.github.com/orgs/wings-software",
    "repos_url": "https://api.github.com/orgs/wings-software/repos",
    "events_url": "https://api.github.com/orgs/wings-software/events",
    "hooks_url": "https://api.github.com/orgs/wings-software/hooks",
    "issues_url": "https://api.github.com/orgs/wings-software/issues",
    "members_url": "https://api.github.com/orgs/wings-software/members{/member}",
    "public_members_url": "https://api.github.com/orgs/wings-software/public_members{/member}",
    "avatar_url": "https://avatars.githubusercontent.com/u/18273000?v=4",
    "description": ""
  },
  "enterprise": {
    "id": 6657,
    "slug": "harness",
    "name": "Harness",
    "node_id": "MDEwOkVudGVycHJpc2U2NjU3",
    "avatar_url": "https://avatars.githubusercontent.com/b/6657?v=4",
    "description": "",
    "website_url": "https://harness.io",
    "html_url": "https://github.com/enterprises/harness",
    "created_at": "2021-04-13T17:56:36Z",
    "updated_at": "2023-02-06T20:48:35Z"
  },
  "sender": {
    "login": "rathodmeetsatish",
    "id": 84321134,
    "node_id": "MDQ6VXNlcjg0MzIxMTM0",
    "avatar_url": "https://avatars.githubusercontent.com/u/84321134?v=4",
    "gravatar_id": "",
    "url": "https://api.github.com/users/rathodmeetsatish",
    "html_url": "https://github.com/rathodmeetsatish",
    "followers_url": "https://api.github.com/users/rathodmeetsatish/followers",
    "following_url": "https://api.github.com/users/rathodmeetsatish/following{/other_user}",
    "gists_url": "https://api.github.com/users/rathodmeetsatish/gists{/gist_id}",
    "starred_url": "https://api.github.com/users/rathodmeetsatish/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/rathodmeetsatish/subscriptions",
    "organizations_url": "https://api.github.com/users/rathodmeetsatish/orgs",
    "repos_url": "https://api.github.com/users/rathodmeetsatish/repos",
    "events_url": "https://api.github.com/users/rathodmeetsatish/events{/privacy}",
    "received_events_url": "https://api.github.com/users/rathodmeetsatish/received_events",
    "type": "User",
    "site_admin": false
  }
}
