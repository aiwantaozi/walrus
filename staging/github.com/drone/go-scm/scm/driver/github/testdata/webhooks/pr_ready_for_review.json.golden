{
  "Action": "review_ready",
  "Repo": {
    "ID": "498312568",
    "Namespace": "wings-software",
    "Name": "meet-git-sync-test",
    "Perm": null,
    "Branch": "main",
    "Private": true,
    "Visibility": 3,
    "Clone": "https://github.com/wings-software/meet-git-sync-test.git",
    "CloneSSH": "git@github.com:wings-software/meet-git-sync-test.git",
    "Link": "https://github.com/wings-software/meet-git-sync-test",
    "Created": "0001-01-01T00:00:00Z",
    "Updated": "0001-01-01T00:00:00Z"
  },
  "PullRequest": {
    "Number": 38,
    "Title": "Update dgdggd.me",
    "Body": null,
    "Sha": "212015fb011c49a2913c4a1a860ce07c0821c362",
    "Ref": "refs/pull/38/head",
    "Target": "main",
    "Source": "rathodmeetsatish-patch-25",
    "Fork": "wings-software/meet-git-sync-test",
    "Link": "https://github.com/wings-software/meet-git-sync-test/pull/38",
    "Diff": "https://github.com/wings-software/meet-git-sync-test/pull/38.diff",
    "Closed": null,
    "Merged": null,
    "Base": {
      "Sha": "8b8d2b3e6a2ee6df108c8429e86c9a750e728f9d",
      "Path": "refs/heads/main",
      "Name": "main"
    },
    "Head": {
        "Sha": "212015fb011c49a2913c4a1a860ce07c0821c362",
        "Path": "refs/heads/rathodmeetsatish-patch-25",
        "Name": "rathodmeetsatish-patch-25"
    },
    "Author": {
      "Login": "rathodmeetsatish",
      "Name": "",
      "Email": "",
      "Avatar": "https://avatars.githubusercontent.com/u/84321134?v=4"
    },
    "Created": "2023-08-28T19:15:53Z",
    "Updated": "2023-08-28T19:16:06Z"
  },
  "Sender": {
    "Login": "rathodmeetsatish",
    "Name": "",
    "Email": "",
    "Avatar": "https://avatars.githubusercontent.com/u/84321134?v=4"
  }
}
